docs/2.8. Usage - Testing Controllers.md

```diff
@@ -14,7 +14,13 @@ In the NMOS Testing Tool select one of the 'Controller' test suites and configur
 
 Select and launch the tests from the NMOS Testing tool, and follow the instructions displayed on the Testing Façade.
 
-### Known Issues
+## Secure Controller Testing
+
+Before running the Controller tests in secure mode, add an entry to the hosts file for 'mocks.testsuite.nmos.tv' pointing at the IP address of the host running the test suite on the host running the test suite and the host running the Controller under test.
```

Comment:
Maybe needs to correspond to config `DNS_DOMAIN`... though we don't currently make that point even in https://github.com/AMWA-TV/nmos-testing/blob/master/test_data/BCP00301/README.md#hosts-files.

Reply:
Hmm, yes. I guess we should get it right here though. Perhaps:

> ...add an entry to the hosts file on the host running the test suite and the host running the Controller under test pointing at the IP address of the host running the test suite. This entry should correspond to the DNS_DOMAIN and be of the form 'mocks.[DNS_DOMAIN]'.  By default this should be 'mocks.testsuite.nmos.tv'.

Reply:
We should probably prefix that with "Secure testing should typically be done in a unicast DNS-SD environment, by setting `DNS_SD_MODE` (see ...)."?

Reply:
Documentation has now ben updated.